docs/mdbook/src/android/map-options-guide.md

```diff
@@ -0,0 +1,308 @@
+# MapLibreMapOptions
+This guide will explain different ways to create a map
+
+When working with maps there are chances of providing default values for MapView to render.
+There are several ways to build MapView:
+1. Providing existing XML namespace tags of MapView in layout
+2. Creating `MapLibreMapOptions` and providing builder function values into MapView
+3. Creating `SupportMapFragment` with the help of `MapLibreMapOptions`
+
+Before explaining MapView configurations we need to know what we can do with both XML namespaces and `MapLibreMaptions`
+
+
+Common configurations can be set:
+- Map base uri
+- Camera
+- Zoom
+- Pitch
+- Gestures
+- Compass
+- Logo
+- Attribution
+- Placement of above elements on the Map and more
+
+We will see how to achieve these configurations in XML layout and programmatically in Activity code step by step
+
+1. Creating MapView namespace and provide some data in layout file
+  ```xml
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please add a comment to each code snippet

```
<!-- this can be found in platform/android/path/to/file -->
```

when it exists somewhere else in the repo.

We'll include the code snippets directly from the source later, but for now a comment like this is OK.